diff --git a/install/install.php b/install/install.php
index aac5745..36053cf 100644
--- a/install/install.php
+++ b/install/install.php
@@ -383,6 +383,10 @@ function step4 ($databasename, $newdatabasename) {
       $query = "UPDATE `glpi_users`
                 SET `language` = NULL";
       $DB->queryOrDie($query, "4203");
+
+      // DEB provides a good system cron
+      $query = "UPDATE `glpi_crontasks` SET `mode`=2 WHERE `name`!='watcher' AND (`allowmode` & 2)";
+      $DB->queryOrDie($query, "4203");
    }
 
    //Check if the port is in url
diff --git a/install/update.php b/install/update.php
index d8584f9..73843b9 100644
--- a/install/update.php
+++ b/install/update.php
@@ -830,6 +830,10 @@ function updateDbUpTo031() {
    $plugin = new Plugin();
    $plugin->unactivateAll();
 
+   // DEB provides a good system cron
+   $query = "UPDATE `glpi_crontasks` SET `mode`=2 WHERE `name`!='watcher' AND (`allowmode` & 2)";
+   $DB->queryOrDie($query);
+
    DBmysql::optimize_tables($migration);
 
    return $ret;
